playbooks/edx_sandbox.yml

```diff
@@ -37,6 +38,10 @@
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
     - { role: 'edxapp', celery_worker: True }
     - edxapp
+    - role: ecommerce
+      when: COMMON_ENABLE_ECOMMERCE
+    - role: ecomworker
+      when: COMMON_ENABLE_ECOMMERCE
```

Comment:
Oops good point, I didn't look closely enough :smile: 

Maybe it would make sense to make the `ecommerce` role depend on the `ecomworker` role. What do you think?